gen_changelog.bat

```diff
@@ -1,7 +1,7 @@
 @echo off
 where npx >nul 2>&1
 
-if %ERRORLEVEL% EQU 0 ( npx auto-changelog -p --sort-commits date --starting-version 0.15.0
+if %ERRORLEVEL% EQU 0 ( npx auto-changelog -p --sort-commits date --starting-version 0.0.0-alpha
```

Comment:
Why?

Reply:
because till 0.15 we created the changelog manualy..

See changelog  you will find
<!-- auto-changelog-above -->

which tells everything above is auto generated

Reply:
+ this repo only has 2 tags so the changelog cant be createed in a good looking manner